docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -281,7 +281,7 @@ public class GreetingResource {
     @GET
     @Produces(MediaType.TEXT_PLAIN)
     @Path("/greeting/{name}")
-    public String greeting(@PathParam String name) {
+    public String greeting(@PathParam("name") String name) {
```

Comment:
It does not if you use the right import. You should use `org.jboss.resteasy.annotations.jaxrs.PathParam` not the JAX-RS one.

Reply:
Ah! Ok. I think I got confused since rest of the annotations come from `javax.ws.rs.*`

Reply:
Yeah, that's why we put the imports in the source excerpts otherwise it's impossible to get it right :).